### DISCUSSION TITLE
Vercel NodeJS Serverless Function not finding local module

### DISCUSSION THREAD

**Discussion by Nathanllee1** [vercel/community#2235] [Help]
### Summary

I'm trying to import a module called "utils.ts" in my root directory from one of my serverless functions in the API directory. I'm getting this error when the serverless function is invoked both locally and on vercel. Interestingly, this setup worked a bit ago, but stopped building with the same setup which has been frustrating.
```
CustomError: Cannot find module '/home/nathanlee/Workspace/Tabify2/utils' imported from /home/nathanlee/Workspace/Tabify2/api/callback.ts
```

I imported the file like this in `callback.ts`
```
import { convertToMySQLDate, query } from "./utils";
```

 In my vercel.json file, I tried using the includeFiles field like this, but it doesn't change whether it's there or not.
```
{
    "api/*.ts": {
       "includeFiles": "./utils.ts" 
    }
}
```

**Comment by maddsua** [comment]
Try importing it as `...utils.js`. Vercel is notorious for making simple thing to shoot you in the foot out of the blue

**Comment by thevinci** [comment]
This shit doesn't work. `includeFiles` and `globs` do nothing. The trick is to make a file in API and just includes it like this:

path.join(process.cwd(), "path/to/file");

Glob doesn't work. You have to list each one individually. I SPENT FUCKING DAYS and after 200 builds TRYING TO GET THIS SHIT TO WORK. FUCKING VERCEL.
